.evergreen/config.yml

```diff
@@ -380,6 +380,20 @@ buildvariants:
     tasks:
       - name: test-aws-lambda-task-group
 
+  - name: happy-eyeballs-macos
```

Comment:
The happy eyeballs test server relies on underspecified TCP handshake behavior that the MacOS (well, BSD kernel) and Windows networking stacks interpret differently than the Linux one, i.e. this test doesn't work on Linux and AFAICT there's no way to make it do so.